README.md

```diff
@@ -34,6 +34,9 @@ The generated tool path can then be saved to a YAML file for use in other applic
 
 ![GUI](docs/gui.png)
 
+To open the Noether GUI application, use the following command:
+`ros2 run noether_gui noether_gui_app`
```

Comment:
Since this package is independent of ROS/ROS2, let's change to something like this instead:


Run the GUI application from its install location in the workspace:
```commandLine
cd <ws>/install/noether_gui/bin
./noether_gui_app
```

Alternatively, run the GUI application using the ROS2 framework:
```commandLine
ros2 run noether_gui noether_gui_app
```

Reply:
Makes sense.